CNI/romana

```diff
@@ -147,13 +145,19 @@ set_up_pod () {
 	KUBEARGS="$KUBEARGS --namespace=$NAMESPACE"
 #	TENANT=$( $KUBECTL $KUBEARGS describe pod $POD | get_labels | get_tenant )
 #	log "--- TENANT = $TENANT ---"
-	SEGMENT=$( $KUBECTL $KUBEARGS describe pod $POD | get_labels | get_segment )
+	SEGMENT=$( $KUBECTL $KUBEARGS get pod $POD -o json | jq -r 'if .metadata.labels["segment"] then .metadata.labels["segment"] else empty end')
 	[[ $SEGMENT ]] || SEGMENT="default"
 	log "--- SEGMENT = $SEGMENT ---"
-	NODE=$( $KUBECTL $KUBEARGS get pods -o wide | grep "$POD " | awk '{ print $6 }' )
+	NODE=$( $KUBECTL $KUBEARGS get pod "$POD" -o json | jq -r '.spec.nodeName')
 	log "--- NODE = $NODE ---"
-	NS_ISOLATION=$( $KUBECTL $KUBEARGS get namespace $NAMESPACE -o json | jq -r '.metadata.annotations["net.alpha.kubernetes.io/network-isolation"]' )
-	[[ $NS_ISOLATION != "on" ]] && NS_ISOLATION="off"
+	NS_ISOLATION="off"
+	NS_POLICY=$( $KUBECTL $KUBEARGS get namespace $NAMESPACE -o json | jq -r '.metadata.annotations["net.beta.kubernetes.io/networkpolicy"] // empty')
```

Comment:
I don't know what's happening here in this block. Maybe my bash is not good enough. Could we add at least a few comments, which explain what the overall idea is here?